integration-tests/src/test/java/oracle/kubernetes/operator/ItJrfPvWlst.java

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+
+import oracle.kubernetes.operator.utils.DbUtils;
+import oracle.kubernetes.operator.utils.DomainCrd;
+import oracle.kubernetes.operator.utils.ExecResult;
+import oracle.kubernetes.operator.utils.JrfDomain;
+import oracle.kubernetes.operator.utils.LoggerHelper;
+import oracle.kubernetes.operator.utils.Operator;
+import oracle.kubernetes.operator.utils.TestUtils;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+
+public class ItJrfPvWlst extends BaseTest {
+  //property file used to customize operator properties for operator inputs yaml
+  private static final String JRF_OPERATOR_FILE_1 = "jrfoperator1.yaml";
+  // file used to customize domain properties for domain, PV and LB inputs yaml
+  private static final String JRF_DOMAIN_ON_PV_WLST_FILE = "jrf_domainonpvwlst.yaml";
+  private static String rcuSchemaPrefix = "jrfdomain";
+  private static Operator operator1;
+  private static String domainNS;
+  private static String domainUid = "";
+  private static String restartTmpDir = "";
+  private static boolean testCompletedSuccessfully;
+  private static String testClassName;
+  private static StringBuffer namespaceList;
+
+  /**
+  * This method gets called only once before any of the test methods are executed. It does the
+  * initialization of the integration test properties defined in OperatorIT.properties and setting
+  * the resultRoot, pvRoot and projectRoot attributes. It also creates Oracle DB pod which used for
+  * RCU.
+  *
+  * @throws Exception - if an error occurs when load property file or create DB pod
+  */
+  @BeforeClass
+  public static void staticPrepare() throws Exception {
+    testClassName = new Object() {
+    }.getClass().getEnclosingClass().getSimpleName();
+    // initialize test properties 
+    initialize(APP_PROPS_FILE, testClassName);  
+  }
+  
+  @Before
+  public void prepare() throws Exception {
+    if (QUICKTEST) {
+      createResultAndPvDirs(testClassName);
+      setMaxIterationsPod(80);
```

Comment:
this can be removed as the global max iterations pod is set to 80 now to give more time for parallel runs

Reply:
Done